dev/devicelab/lib/tasks/web_dev_mode_tests.dart

```diff
@@ -45,21 +45,11 @@ TaskFunction createWebDevModeTest(String webDevice, bool enableIncrementalCompil
           final Process packagesGet = await startProcess(
               path.join(flutterDirectory.path, 'bin', 'flutter'),
               <String>['packages', 'get'],
-              environment: <String, String>{
-                'FLUTTER_WEB': 'true',
-                if (enableIncrementalCompiler)
-                  'WEB_INCREMENTAL_COMPILER': 'true',
-              },
           );
           await packagesGet.exitCode;
           final Process process = await startProcess(
               path.join(flutterDirectory.path, 'bin', 'flutter'),
               flutterCommandArgs('run', options),
-              environment: <String, String>{
-                'FLUTTER_WEB': 'true',
-                if (enableIncrementalCompiler)
```

Comment:
Do we still need the enableIncrementalCompiler checks?

Reply:
Nope, removed

Reply:
these were for the frontend_server based web compilation, but that is on by default now